docs/data/toolpad/components/button.md

```diff
@@ -0,0 +1,55 @@
+# Button
+
+<p class="description">Learn about a button component and it's usage in Toolpad.</p>
+
+Buttons allow users to take actions, make choices with a single tap. Detailed documentation of the component properties is available in the reference section for [button](/toolpad/reference/components/button/#properties).
+
+{{"demo": "ButtonBasic.js", "hideToolbar": true}}
```

Comment:
This button here looks different from the buttons in Toolpad, as it's probably using the theme from the docs.